database/redis/selfstate.go

```diff
@@ -3,6 +3,7 @@ package redis
 import (
 	"github.com/gomodule/redigo/redis"
 	"github.com/moira-alert/moira"
+	"github.com/moira-alert/moira/api/dto"
```

Comment:
I do not like usage of `dto` package in database layer. the only reason of DTO is a conversion to JSON and from JSON in API.

Reply:
It was only using a variable from the `dto` package, I've changed it to remove the import now.